json-logs/samples/api/canvases.create.json

```diff
@@ -0,0 +1,11 @@
+{
+  "ok": false,
+  "canvas_id": "F016BFXUEQ2",
+  "error": "",
```

Comment:
document says "detail" can exist too when the error is returned for canvases.create|edit